dragos-worldview-threat-intelligence/info.json

```diff
@@ -322,10 +323,10 @@
             "ASC",
```

Comment:
Please change the options value to ["Ascending", "Descending"]. Also, make changes wherever you have use Sort Order parameter.
Note: Handle those values in constants.py file

Reply:
Done.